runtime/command-version/src/main/java/io/aklivity/zilla/runtime/command/version/internal/airline/ZillaVersionCommand.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc.
+ *
+ * Aklivity licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.aklivity.zilla.runtime.command.version.internal.airline;
+
+import com.github.rvesse.airline.annotations.Command;
+
+import io.aklivity.zilla.runtime.command.ZillaCommand;
+
+@Command(name = "version", description = "Display version information")
+public final class ZillaVersionCommand extends ZillaCommand
+{
+    public ZillaVersionCommand()
+    {
+    }
+
```

Comment:
public no-args constructor is implicit, please remove explicit public no-args constructor.